quattor/types/hardware.pan

```diff
@@ -20,9 +20,12 @@ type structure_ram = {
 }
 type structure_cpu = {
     include structure_annotation
-    "speed"  : long # "CPU clock speed in MHz"
-    # Number of cores on each CPU chip, defaults to 1.
+    @{ desc = CPU clock speed in MHz }
+    "speed"  : long
+    @{ desc = Number of cores on each CPU chip }
     "cores" : long(1..) = 1
+    @{ desc = Number of execution threads on each CPU chip }
```

Comment:
you mean `each CPU core`.
what is the relation with the hyperthreading option? i'm assuming this option is the number of SMT threads
also, what's up with the `desc =`? all annotations like this are descriptions (for functions, it makes sense to distinguish between args and description